recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "kwant" %}
+{% set py3k_ver = "1.2.2" %}
+{% set py2k_ver = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ py3k_ver }}  # [py3k]
+  version: {{ py2k_ver }}  # [py2k]
```

Comment:
Why do we have 2 different versions? Could we not just start with 1.1.2 and then drop Python 2.x support when upgrading or is there more going on here?